packages/block-library/src/post-author/index.php

```diff
@@ -40,12 +40,6 @@ function post_author_build_css_colors( $attributes ) {
 	$has_named_background_color  = array_key_exists( 'backgroundColor', $attributes );
 	$has_custom_background_color = array_key_exists( 'customBackgroundColor', $attributes );
 
-	// If has background color.
-	if ( $has_custom_background_color || $has_named_background_color ) {
-		// Add has-background-color class.
-		$colors['css_classes'][] = 'has-background-color';
-	}
```

Comment:
Why is this removed?

Reply:
It causes themes to overwrite the font color that gets set and saved.  Themes apply styles to this to selector to set color on the font to ensure it is still readable.  If we are also controlling the font color, then it is unnecessary as well.  Considering it more, we should re-apply this class name, but only when a font color is not set.

Reply:
> Considering it more, we should re-apply this class name, but only when a font color is not set.

Or should we?  In that case if someone adds this block and changes the background to a very light color (without changing any font color), a theme would still then force their font color to be really light as well.

Reply:
Maybe the best solution is to add this class back in and move the font color styles that get set to inline styles.

Reply:
We should mirror what the other core blocks are doing.

Reply:
Ok, I re-added that class and moved the font color classes/styles into the content div instead.